packages/cozy-mespapiers-lib/src/components/Papers/CategoryItemByKonnector.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { useQuery, isQueryLoading } from 'cozy-client'
+import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+
+import { buildAccountsQueryBySlug } from '../../helpers/queries'
+import { useScannerI18n } from '../Hooks/useScannerI18n'
+import { addAccountsToKonnectors, makeIsLast } from './helpers'
+import styles from './styles.styl'
+
+const CategoryItemByKonnector = ({
+  konnectorsWithAccounts,
+  konnector,
+  onClick
+}) => {
+  const scannerT = useScannerI18n()
+
+  const queryAccounts = buildAccountsQueryBySlug(konnector.slug)
+  const { data: accounts, ...accountsQueryLeft } = useQuery(
+    queryAccounts.definition,
+    queryAccounts.options
+  )
+  const isLoading = isQueryLoading(accountsQueryLeft)
```

Comment:
be careful if you have a fetch policy on this query. You should rely on `isQueryLoading` AND `hasQueryBeenLoaded`

Reply:
oui ok pour éviter de retomber dans le isLoading si jamais on retourne sur cette page et que la fetchpolicy a expiré (si j'ai bien compris)